umpost/um2netcdf.py

```diff
@@ -70,6 +70,14 @@
 LEVEL_HEIGHT = "level_height"
 SIGMA = "sigma"
 
+# forecast reference time constants
+FORECAST_REFERENCE_TIME = "forecast_reference_time"
+TIME = "time"
+PROLEPTIC_GREGORIAN = "proleptic_gregorian"
+GREGORIAN = "gregorian"
+FORECAST_PERIOD = "forecast_period"
+STANDARD = "standard"
+
 
```

Comment:
We might be able to use the `cf_units` calendar constants that are used by `def pg_calendar` if it's preferable. E.g. `cf_units.CALENDAR_PROLEPTIC_GREGORIAN`, `cf_units.CALENDAR_GREGORIAN`, and `cf_units.CALENDAR_STANDARD`.

Reply:
Indeed, using the `cf_units` constants is preferable to duplicating them. The fixes are in with 7ee59a23.